### PR TITLE
add new codelab: rabbitmq-shovel-to-solace

### PR DESCRIPTION
**Pull request by LeahRobert**
RabbitMQ Shovel to Solace - initial

### CODE REVIEW

**Comment by Mrc0113** [comment]
Thanks for opening up a PR and contributing to our open source codelabs! 

## Review process Guidelines
Once the PR is opened, tag **at least one** Technical Reviewer

### Technical Reviewer Responsibilities 
- [ ] Dry-run technical steps 
- [ ] Review technical positioning/messaging
- [ ] Point out typos and potential formatting issues
- [ ] Technical Reviewer(s): "Approve" or "Request Changes" by navigating to the `Files changed` tab and click on the "Review Changes" drop down    
Review Changes button

### Post Technical Review
A member of the Developer Advocate Team will review overall structure and merge into master which will push it to production

File: markdown/rabbitmq-shovel-to-solace/rabbitmq-shovel-to-solace.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
+author: Leah Robert
+summary: Bridge data from RabbitMQ to Solace PubSub+ with the RabbitMQ Shovel plugin
+id: solace-rabbit-shovel
+tags: workshop, amqp, rabbit
```
Comment (Mrc0113) [inline markdown/rabbitmq-shovel-to-solace/rabbitmq-shovel-to-solace.md:4]: change to `iguide`

File: markdown/rabbitmq-shovel-to-solace/rabbitmq-shovel-to-solace.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+categories: Solace,Java,Spring,Javascript,Aurelia,TypeScript
```
Comment (Mrc0113) [inline markdown/rabbitmq-shovel-to-solace/rabbitmq-shovel-to-solace.md:5]: Change to `Solace,RabbitMQ`

File: markdown/rabbitmq-shovel-to-solace/rabbitmq-shovel-to-solace.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+environments: Web
+status: Published
+feedback link: http://github.com/solacetraining/solace-battleship/issues
```
Comment (Mrc0113) [inline markdown/rabbitmq-shovel-to-solace/rabbitmq-shovel-to-solace.md:8]: Change to `https://solace.community` if nowhere else to point them to.

**Comment by LeahRobert** [comment]
@Himanshu Gupta
@TKTheTechie 
Could I ask either of you, or both, to review this codelab?

**Review by TamimiGitHub — DISMISSED** [review]

**Review by TamimiGitHub — APPROVED** [review]

**Pull request merged**